packages/database/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
set it to private: true

Reply:
Again maybe yes, but we might want to publish in npm at some point

Reply:
do we? this is a useless package for anyone else and there's no point with having to deal with package versions/publishing/whatnots, i'd just bundle it with web and api production builds and palantir